plenum/server/replicas.py

```diff
@@ -0,0 +1,132 @@
+from collections import deque
+from typing import List, Generator
+
+from plenum.server.monitor import Monitor
+from plenum.server.replica import Replica
+from stp_core.common.log import getlogger
+
+logger = getlogger()
+
+MASTER_REPLICA_INDEX = 0
+
+
+class Replicas:
+
+    def __init__(self, node, monitor: Monitor):
+        # passing full node because Replica requires it
+        self._node = node
+        self._monitor = monitor
```

Comment:
Why make Monitor a dependency of `Replicas`, when f changes let the Node communicate to Monitor and Replicas? Node is the orchestrator.